CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Nautobot Secrets Providers Changelog
 
+## v1.3.0 (2022-08-29)
+
+### Added
+
+- [#32](https://github.com/nautobot/nautobot-plugin-secrets-providers/issues/32) Add support for skipping certificate validation when connecting to HashiCorp Vault.
+- [#34](https://github.com/nautobot/nautobot-plugin-secrets-providers/issues/34) Add support for alternate authentication to HashiCorp Vault via AWS and Kubernetes authentication methods.
+- [#38](https://github.com/nautobot/nautobot-plugin-secrets-providers/pull/38) Add support for Python 3.10.
+- [#40](https://github.com/nautobot/nautobot-plugin-secrets-providers/issues/40) Add `default_mount_point` config option for HashiCorp Vault.
+
+### Changed
+
+- [#42](https://github.com/nautobot/nautobot-plugin-secrets-providers/issues/42) Now requires python-tss-sdk version v1.2 or later
```

Comment:
Replace 'requires' with 'supports', please.

I've added the new import statement in delinea.py but left the old (thycotic) import statement unchanged. The API itself has not changed so it works for both

Reply:
We would have to revert the version bump in this PR:

https://github.com/nautobot/nautobot-plugin-secrets-providers/pull/44/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R33

As of right now this will require updating `python-tss-sdk`.